projects/packages/search/src/dashboard/components/record-meter/barchart.jsx

```diff
@@ -0,0 +1,137 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+const Chart = React.lazy( () => import( 'chart.js/auto' ) );
+
+/**
+ * Internal dependencies
+ */
+
+export class BarChart extends React.Component {
+	constructor( props ) {
+		super( props );
+		this.canvasRef = React.createRef();
+	}
+
+	componentDidMount() {
+		this.myChart = new Chart( this.canvasRef.current, {
+			type: 'bar',
+			options: {
+				borderRadius: 100,
+				borderSkipped: 'middle',
+				indexAxis: 'y',
+				maintainAspectRatio: false,
+				aspectRatio: 1.3,
+				scales: {
+					x: {
+						stacked: true,
+						grid: {
+							display: false,
+							drawBorder: false,
+						},
+						ticks: {
+							display: false,
+						},
+					},
+					y: {
+						stacked: true,
+						grid: {
+							display: false,
+							drawBorder: false,
+						},
+						ticks: {
+							display: false,
+						},
+					},
+				},
+
+				plugins: {
+					title: {
+						display: false,
+					},
+					subtitle: {
+						display: false,
+					},
+					legend: {
+						display: false,
+
+						labels: {
+							filter: function ( legendItem, data ) {
+								return ! legendItem.text.includes( 'Remaining' );
+							},
+						},
+					},
+				},
+			},
+
+			data: {
+				labels: [ '' ], // this empty label must remain, if removed the entire chart breaks
+				datasets: this.props.data.map( d => d.data ),
+			},
+		} );
+
+		if ( this.props.isValid === true ) {
+			this.setState( { legendItems: this.myChart.legend.legendItems } );
+		}
+	}
+
+	render() {
+		if ( this.props.isValid === false ) {
+			return null;
+		}
+		return (
+			<div className="barChart">
+				<div className="chartContainer">
+					<canvas ref={ this.canvasRef }>
+						<p>Text alternative for this canvas graphic is in the data table below.</p>
+						<table
+							border="0"
+							cellPadding="5"
+							summary="This is the text alternative for the canvas graphic."
+						>
+							<caption>Records Indexed for Jetpack Search</caption>
+							<tbody>
+								<tr>
+									<th scope="col">Post Type</th>
```

Comment:
We need to translate these strings using something like `__( ' Post Type', 'jetpack-search-pkg' )`.